invisible_cities/reco/corrections_test.py

```diff
@@ -154,35 +155,40 @@ def test_correction_attributes_2d(toy_data_2d):
 
 @given(uniform_energy_2d())
 def test_correction_attributes_2d_unnormalized(toy_data_2d):
-    X, Y, _, _, F, Fu, _, _, correct = toy_data_2d
+    X, Y, _, _, F, Fu, _, _ = toy_data_2d
     c = Correction((X,Y), F, Fu, False)
     assert_allclose(c._fs, F )
     assert_allclose(c._us, Fu)
 
 
 @given(uniform_energy_2d())
 def test_correction_call_2d(toy_data_2d):
-    X, Y, _, _, F, Fu, _, _, correct = toy_data_2d
+    X, Y, E, Eu, F, Fu, i_max, j_max = toy_data_2d
+    interp_strategy="nearest"
+    correct = Correction((X,Y), E, Eu,
+                           norm_strategy = "index", index = (i_max, j_max),
+                         interp_strategy = interp_strategy)
 
-    # Combine x and y so they are paired
-    X_test = np.repeat(X, Y.size)
-    Y_test = np.tile  (Y, X.size)
-    F_test, U_test = correct(X_test, Y_test)
-    assert_allclose(F_test, F )
-    assert_allclose(U_test, Fu)
+    # create a collection of (x,y) point such that the
+    # x coordinates are stored in X_sample and the y coordinates in Y_sample
```

Comment:
Just so I can keep my training as a padawan... why is this better than using np.repeat and np.tile?